packages/core/content-type-builder/admin/src/components/Relation/RelationNaturePicker/RelationNaturePicker.tsx

```diff
@@ -12,15 +10,14 @@ import {
 import get from 'lodash/get';
 import truncate from 'lodash/truncate';
 import pluralize from 'pluralize';
-import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 import { useDispatch } from 'react-redux';
 
-import useDataManager from '../../../hooks/useDataManager';
-import getTrad from '../../../utils/getTrad';
+import { useDataManager } from '../../../hooks/useDataManager';
+import { getTrad } from '../../../utils/getTrad';
 import { ON_CHANGE_RELATION_TYPE } from '../../FormModal/constants';
 
-import { IconWrapper, InfosWrapper, Wrapper } from './components';
+import { IconWrapper, InfosWrapper, Wrapper } from './Components';
```

Comment:
```suggestion
import { IconWrapper, InfosWrapper, Wrapper } from './components';
```
But keep in mind that the name of the file `components.tsx` will throw an eslint error due to the PascalCase rule